test/unit/common/Reducers.test.js

```diff
@@ -723,11 +711,6 @@ describe("Reducers", () => {
       const newState = DiscoveryStream(undefined, {type: at.PLACES_LINK_BLOCKED});
       assert.equal(newState, INITIAL_STATE.DiscoveryStream);
     });
-    it("should not update state if feeds are not loaded", () => {
-      const deleteAction = {type: at.PLACES_LINK_BLOCKED, data: {url: "foo.com"}};
```

Comment:
this is changed with spoc_fills patch

Reply:
That patch changes a lot of things for me :D

Reply:
Not sure why I deleted this test, it looks unrelated and seems to pass just fine. Possibly a mistake of some sort. Adding it back.